### PR TITLE
Feature/toggle sidebar

### PR DESCRIPTION
**Pull request by davidenke**
This shows the deployed PR as well ;)

### CODE REVIEW

File: src/components/layout/aside/aside.component.scss [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 // prettier-ignore
 :host {
-  ---wcp-aside-collapsed-width: var(--wcp-aside-collapsed-width, #{utils.size(5)});
-  ---wcp-aside-expanded-width: var(--wcp-aside-expanded-width, #{utils.size(24)});
+  ---wcp-aside-max-width: var(--wcp-aside-max-width, #{utils.size(24)});
```
Comment (TomKopp) [inline src/components/layout/aside/aside.component.scss:5]: All custom properties have three dashes ('---') at the start not just two.
Is there a reason for that?
I have missed that on the other elements.

**Comment by TomKopp** [comment]
I would second the Github Action bot with its warning.
'tsdoc-unnecessary-backslash: A backslash can only be used to escape a punctuation character'
For a line-break you could use three white-spaces.

File: src/components/feature/toggle-sidebar/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# wcp-toggle-sidebar
+
+Shows a button to toggle sidebar.
+
+## Example
+
+```html
+<wcp-wcp-toggle-sidebar></wcp-wcp-toggle-sidebar>
```
[the remark this answers is not in the document]
Comment (davidenke) [reply src/components/feature/toggle-sidebar/README.md:8]: Yes, double feature! No, that's addressed in the other PR already (if you don't mind)...

File: src/components/layout/aside/aside.component.scss (hunk 1 above)
Comment (davidenke) [reply src/components/layout/aside/aside.component.scss:5]: This should be this way in _every_ component! The idea is to allow theming in a convenient place somewhere up the DOM (e.g. `:root`) and thus, inheriting **public props** (_two dashes!_) in every component. So the usage inside the component is mapped (with defaults) to the **internal props** (_triple dashes!_).

Comment (TomKopp) [reply src/components/layout/aside/aside.component.scss:5]: Ah ok, didn't know that 👍

**Pull request merged**